res/script/sdbSpecList.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#
+# This shell script print tizen device serial name on each line.
+#
+
+sdb devices | grep -v devices | grep device | awk '{print $1}'
```

Comment:
is it okay for save this script on this position?

Reply:
Currently `res` contains some test materials.
I guess that it would be possible to exclude `res` from files list to be packaged.

IMHO, how about `{workspace_folder}/script`?
Also we have to make sure that `{workspace_folder}/script` should be packaged.

Reply:
Can I ask a favor of you?
For readers of this code to understand this script, could you add comment showing what the output of `$ sdb devices` looks like ?